bins/src/bin/datadog-static-analyzer-server.rs

```diff
@@ -51,6 +51,38 @@ impl Fairing for CORS {
     }
 }
 
+pub struct CustomHeaders;
+
+#[rocket::async_trait]
+impl Fairing for CustomHeaders {
+    fn info(&self) -> Info {
+        Info {
+            name: "Custom Headers",
+            kind: Kind::Response,
+        }
+    }
+
+    async fn on_response<'r>(&self, request: &'r RocketRequest<'_>, response: &mut Response<'r>) {
+        let state = request.guard::<&State<ServerConfiguration>>().await;
+
+        if let rocket::outcome::Outcome::Success(server_configuration) = state {
+            response.set_header(Header::new(
+                "X-static-analyzer-server-shutdown-enabled",
```

Comment:
Put the flag as a constant in `constants.rs` as `SERVER_HEADER_SHUTDOWN_ENABLED` so all headers (either, request or response) are in one place.

Reply:
Added them to `server/constants.rs`.